karapuzio/assignment2/q1_ancestors/BinaryTree.java

```diff
@@ -0,0 +1,135 @@
+package edu.codeU.assignment2.Ancestors;
+
+import edu.codeU.assignment2.BinarySearchElementExeption;
+import edu.codeU.assignment2.BinaryTreeInitializationExceprion;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+
+public class BinaryTree{
```

Comment:
Can you try and define one BinaryTree class and use it for both problems? Not only will it simplify my life while reviewing your code but also potentially yours when you keep working on your code.